Vizzuality/waterbird-population
# Feature/data carto

This PR fetches families from CARTO table

## Testing instructions

Go to explore section and make, all families should appear collapsed and when you expand them you should be able to see static details for each one.
